plugin.xml

```diff
@@ -60,8 +60,8 @@
 		<source-file src="src/android/src/com/pushwoosh/plugin/internal/PhonegapPluginProvider.java"
 			target-dir="src/com/pushwoosh/plugin/internal" />
 
-		<framework src="com.google.firebase:firebase-core:16.0.4" />
-		<framework src="com.google.firebase:firebase-messaging:17.3.4" />
+		<framework src="com.google.firebase:firebase-core:(+,17.0.0)" />
+		<framework src="com.google.firebase:firebase-messaging:(+,19.0.0)" />
```

Comment:
Is this safe? This can cause AndroidX related issues (I assume it does not since the app compiled in order to do tests).

But it can cause conflicts with the Firebase Plugin from the Experts, we already had a support case with this issue. Please make sure both plugins can coexist.

Reply:
@EiyuuZack don't forget to check this!

Reply:
Indeed there were conflicts with Firebase Plugin. Question here is whether we should be the ones to accommodate an unsupported plugin or they should comply with the service.

In the interest of time, because it is a given this will not be addressed in the short term, the dependencies were downgraded to match Firebase's.

Reply:
I agree, we just did change that to close a high priority support case.